### PR TITLE
[ru] libmanga fix 18+ mangas w/o authorization

### PR DESCRIPTION
**Pull request by Pavkazzz**

### CODE REVIEW

File: src/ru/libmanga/src/eu/kanade/tachiyomi/extension/ru/libmanga/LibManga.kt [hunk 1]
```diff
@@ -250,38 +250,17 @@ class LibManga : ConfigurableSource, HttpSource() {
...
+        val json = jsonParser.parse(body)
+        val images = json["images"].array
+        val chapterslug = json["chapter"]["slug"].string
 
         val pages = mutableListOf<Page>()
-        pagesJson.forEach { page ->
-            pages.add(Page(page["p"].int, "", imageServerUrl + imgUrl + page["u"].string))
+        images.forEachIndexed { index, page ->
+            val url = "${this.imageServerUrl()}/manga/$slug/chapters/$chapterslug/${page.string}"
+            Log.i("LIB", "$index + $url")
```
Comment (SnakeDoc83) [inline src/ru/libmanga/src/eu/kanade/tachiyomi/extension/ru/libmanga/LibManga.kt:262]: Could you remove the `Log` and its import?

Comment (Pavkazzz) [reply src/ru/libmanga/src/eu/kanade/tachiyomi/extension/ru/libmanga/LibManga.kt]: Thx for review!

**Comment by Pavkazzz** [comment]
> Also, could you take a look at a couple open issues for this source?
> 
> #4210
> #2601
> 
> I'm not sure if this PR addresses the first one. You don't have to do anything for them on this PR, but if you could eventually get to them that'd be great. If you disagree with the name change, tell me to close it.

Fix - https://github.com/inorichi/tachiyomi-extensions/issues/4210

https://github.com/inorichi/tachiyomi-extensions/issues/2601 - It's the main reason for this patch. Resolved too

**Comment by SnakeDoc83** [comment]
Thanks!

**Pull request merged**